### PR TITLE
feat(svg-icon): enable public assets url configuration

### PR DESCRIPTION
**Pull request by djcouto**
#### Description
Currently, the links to the SVG icons are relative to the page, which implies the SVG icons file to be served on every relative page where it's running.
This PR enables this public assets URL to be configured via prop and also defaults it to be absolute.

### CODE REVIEW

File: packages/ui-components/src/components/svg-icon/readme.md [hunk 1]
```diff
@@ -100,22 +101,6 @@ export const SvgIconExample: React.FC = () => (
 | `"icon"` | The icon container. |
 
 
-## CSS Custom Properties
```
Comment (ricardogoncalves89) [inline packages/ui-components/src/components/svg-icon/readme.md:103]: why is this removed?

**Review by ricardogoncalves89 — CHANGES_REQUESTED** [review]

File: packages/ui-components/src/components/svg-icon/readme.md (hunk 1 above)
Comment (djcouto) [reply packages/ui-components/src/components/svg-icon/readme.md:103]: Ups, don't know, I didn't notice, probably something went wrong when generating the readme.

I will reset this on the upcoming commit.

**Review by ricardogoncalves89 — APPROVED** [review]

**Review by pedrorebelocosta — APPROVED** [review]
